tests/test_labeling.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (C) 2019-2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+import os.path as osp
+from unittest.case import TestCase
+
+import numpy as np
+
+from datumaro.components.annotation import AnnotationType, Label, LabelCategories
+from datumaro.components.extractor import DatasetItem
+from datumaro.components.media import Image
+from datumaro.components.project import Dataset
+
+from .requirements import Requirements, mark_requirement
+
+TMP_DATASET_DIR = osp.join(osp.dirname(__file__), "tmp", "test_labeling")
```

Comment:
We can use `tempfile.TemporaryDirectory` for temporary usage (automatically reserve and clean up resources at code level). Please refer to https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory.

Reply:
cool!